src/__tests__/screen.js

```diff
@@ -1,6 +1,10 @@
 import {screen} from '..'
 import {renderIntoDocument} from './helpers/test-utils'
 
+jest.mock('../get-user-code-frame', () => ({
+  getUserCodeFrame: () => '',
+}))
+
```

Comment:
Why is this mocked here and not all the other tests? What's special about this one?

Reply:
Given that the test `exposes debug method` internally calls `logDOM`, which in turn calls `getUserCodeFrame`, we need to mock it so it doesn't add clutter to the output of the screen's tests snapshots

Reply:
Gotcha. Let's write a comment explaining that please. Thanks!

Reply:
Done :)